internal/services/projects_test.go

```diff
@@ -0,0 +1,281 @@
+package services
+
+import (
+	"testing"
+
+	"github.com/checkmarx/ast-cli/internal/wrappers"
+	"github.com/checkmarx/ast-cli/internal/wrappers/mock"
+	"github.com/spf13/cobra"
+)
+
+func TestFindProject(t *testing.T) {
+	type args struct {
+		applicationID           []string
+		projectName             string
+		cmd                     *cobra.Command
+		projectsWrapper         wrappers.ProjectsWrapper
+		groupsWrapper           wrappers.GroupsWrapper
+		accessManagementWrapper wrappers.AccessManagementWrapper
+		applicationsWrapper     wrappers.ApplicationsWrapper
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    string
+		wantErr bool
+	}{
+		// TODO: Add test cases.
```

Comment:
same here

Reply:
It was auto-generated comment Goland to guide the developer will see where to add the test data.
I'll remove it.